cmd/security-secretstore-setup/README.md

```diff
@@ -0,0 +1,39 @@
+# EdgeX Foundry Security Service - Security Secretstore Setup
+[![license](https://img.shields.io/badge/license-Apache%20v2.0-blue.svg)](LICENSE)
+
+Go implementation of EdgeX security-secretstore-setup service.
+
+## Build
+
+Use the Makefile in the root directory of the repository to build  security-secrets-setup:
+
+```sh
+ make cmd/security-secretstore-setup/security-secretstore-setup
+```
+
+This will create an executable located at `cmd/security-secretstore-setup/` if successful.
+
+## Run security-secretstore-setup with different parameters
+
+The binary supports multiple command line parameters 
+
+```sh
+ --init  //run init procedure for security service
+ --insecureSkipVerify //skip server side SSL verification, mainly for self-signed cert
+ --configfile //use different configuration file other than the default
+ --vaultInterval //time to wait between checking Vault status in seconds
+```
+
+An example of use of the parameters can be found from docker compose file
```

Comment:
s/An example of use of the parameters can be found from docker compose file/An example of using the parameters can be found in the docker compose file

Reply:
fixed.